modules/QtPDF/trans/QtPDF_it.ts

```diff
@@ -759,7 +759,8 @@
     <message>
         <location filename="../src/PDFDocumentView.cpp" line="34"/>
         <source>[translator&apos;s name/email]</source>
-        <translation>trebestie@gmail.com</translation>
+        <translation>trebestie@g<translation>
+        <translation>albano_battistella@hotmail.com</translation>
```

Comment:
Qt doesn't seem to like more than one translation for any given item. Please merge the two (e.g. by a comma separated list ("PersonA, PersonB") or by a translated "and" ("PersonA e PersonB")).